includes/modules/pages/login/jscript_paypal_incontext.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Javascript enabler for providing and handling the PayPal 
+ *   Express In Context shopping experience.
+ *
+ * @package     paypal_incontext
+ * @copyright   Copyright 2003-2016 Zen Cart Development Team
+ * @copyright   Portions Copyright 2003 osCommerce
+ * @copyright   Portions Copyright 2012-2016 mc12345678
+ * @license     http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: Author: mc12345678  Wed Aug 24 20:20:09 2016 -0500 Modified in v1.6.0 $
+ */
+ 
+    $buttonContainer = 'ecicButton';
```

Comment:
Causes a javascript error when page is accessed prior to the cart having contents. ($ec_enabled variable or similarly named not providing a false response)

Updating to identify the button the same way the template code is written or '' if not true. All within a line instead of a formal if/else construction.